src/features/form/layout/PageNavigationContext.tsx

```diff
@@ -101,3 +112,14 @@ export const useSetReturnToView = () => {
   const func = useLaxSelector((ctx) => ctx.setReturnToView);
   return func === ContextNotProvided ? undefined : func;
 };
+
+export const useSummaryNodeOfOrigin = (): BaseLayoutNode<CompSummaryInternal> | undefined => {
```

Comment:
```suggestion
export const useSummaryNodeOfOrigin = (): LayoutNode<'Summary'> | undefined => {
```

Super-minor nitpick.. :wink: Saying that the type is `BaseLayoutNode` means that "the node type for the Summary component is always the base class", but technically each component type can override the BaseLayoutNode class with their own functionality. That's not in use right now, but it might soon be in a task I'm working on, and I had to look around for all the direct references to `BaseLayoutNode` and replace them with `LayoutNode` (which is the type you _should_ use, as that will automatically figure out the actual class).